doc/source/examples/JobAnalysis/job_analysis.ipynb.rst

```diff
@@ -228,16 +236,16 @@ In addition to the fluent syntax, both dictionary and dot syntaxes are also supp
-…       …     …                               …                 …                 …         …                    …                 …             …                 …                     …               …                               …                               …
-adf.016 True  None                            CH4O              CO                20.258706 GeometryOptimization TZP               PBE           None              None                  -2900.38        [‘C’, ‘O’, ‘H’, ‘H’, ‘H’, ‘H’]  [ 0.58673094 -0.60299606 -0.10… [array([-0.36298962, -0.021487…
-neb     False NEB optimization did NOT conve… : CHN, final: CHN : C=N, final: C#N 0.523557  NEB                  None              None          9                 10                    None            [‘C’, ‘N’, ‘H’]                 None                            [array([0.46287785, 0.20144834…
-neb.002 True  None                            : CHN, final: CHN : C=N, final: C#N 0.869299  NEB                  None              None          9                 100                   -14936.53       [‘C’, ‘N’, ‘H’]                 [-0.00728359 -0.21164112 0.21…  [array([0.56171873, 0.20547696…
-======= ===== =============================== ================= ================= ========= ==================== ================= ============= ================= ===================== =============== =============================== =============================== ===============================
+======= ===== =============================== ================= ================= ========= ================== ==================== ================= ============= ================= ===================== =============== =============================== =============================== ===============================
+Name    OK    ErrorMsg                        Formula           Smiles            CPUTime   ams:General%engine InputAmsTask         InputAdfBasisType InputAdfXcGga InputAmsNebImages InputAmsNebIterations Energy [kJ/mol] AtomType                        Charge                          AtomCoords
+======= ===== =============================== ================= ================= ========= ================== ==================== ================= ============= ================= ===================== =============== =============================== =============================== ===============================
+dftb    True  None                            C2H6              CC                0.189760  dftb               SinglePoint          None              None          None              None                  -19594.01       [‘C’, ‘C’, ‘H’, ‘H’, ‘H’, ‘H’,… [-0.07293185 -0.07372966 0.02…  [array([-0.74763668, 0.041837…
```

Comment:
The extra dftb column is maybe a bug?

Reply:
Same, new column is `ams:General%engine` i.e. from `.add_rkf_field("General", "engine")`